packages/shared/src/components/cards/article/ArticleList.tsx

```diff
@@ -124,7 +125,8 @@ export const ArticleList = forwardRef(function ArticleList(
                 onShare={onShare}
                 post={post}
                 imageProps={{
-                  loading: 'lazy',
+                  loading: eagerLoadImage ? 'eager' : 'lazy',
+                  fetchPriority: eagerLoadImage ? 'high' : 'auto',
```

Comment:
Same as previous comment:
https://github.com/dailydotdev/apps/pull/3673/files#r1800749778

Reply:
Solved adding the expect error only in Image component :)